src/service/action-impl.js

```diff
@@ -152,11 +158,40 @@ export class ActionService {
           this.trigger(dev().assertElement(event.target), 'tap', event);
         }
       });
-    } else if (name == 'submit' || name == 'change') {
+    } else if (name == 'submit') {
       this.root_.addEventListener(name, event => {
         this.trigger(dev().assertElement(event.target), name, event);
       });
+    } else if (name == 'change') {
+      this.root_.addEventListener(name, event => {
+        const target = event.target;
+        const changeEvent = this.getChangeDetails_(event);
```

Comment:
I highly recommend adding a `detail` property to the existing `event` object instead of cloning it into a custom event. Components that register actions may need access to the original event.